packages/hoppscotch-sh-admin/src/helpers/userManagement.ts

```diff
@@ -38,21 +27,22 @@ const displayToastMessages = (
   });
 };
 
-export const handleUserDeletion: HandleUserDeletion = (
-  deletedUsersList,
-  metadata
-) => {
+export const handleUserDeletion = (deletedUsersList: UserDeletionResult[]) => {
   const uniqueErrorMessages = new Set(
     deletedUsersList.map(({ errorMessage }) => errorMessage).filter(Boolean)
   ) as Set<string>;
 
   const isBulkAction = deletedUsersList.length > 1;
 
+  const deletedUserIDs = deletedUsersList
+    .filter((user) => user.isDeleted)
+    .map((user) => user.userUID);
+
   // Show the success toast based on the action type if there are no errors
   if (uniqueErrorMessages.size === 0) {
     if (isBulkAction) {
       toast.success(
-        (metadata as BulkActionMetadata).areMultipleUsersSelected
+        isBulkAction
           ? t('state.delete_user_success')
           : t('state.delete_users_success')
       );
```

Comment:
This was already within the `isBulkAction` block; updated in https://github.com/hoppscotch/hoppscotch/commit/787aab650ff584f9d9cd86aea8935c74b72bc632.